src/test/resources/inputFileTest.csv

```diff
@@ -0,0 +1,3 @@
+type,fruit,quantity
+b,banana,20
+b,apple,100
```

Comment:
Why in `inputFileForReport.csv` do you have an empty line at the end, but not here?

Reply:
fixed